Automattic/wp-calypso
# Business Tools redesign: Update plugins banner

Resolves https://github.com/Automattic/dotcom-forge/issues/9657

## Proposed Changes

* Update the banner for plugins under `/marketing/tools/:siteId` and `/sites/marketing/tools/:siteId`

|View|Before | After|
|-------|-------|------|
| Sites     |  ![CleanShot 2024-11-08 at 19 04 29@2x](https://github.com/user-attachments/assets/9ecc9182-4a0b-4a41-b794-cb29d4ba3084)  |   ![CleanShot 2024-11-08 at 18 58 08@2x](https://github.com/user-attachments/assets/07cc61d4-3a81-4f19-90b7-fe2b216f18ea)      |
|  Home    |  ![CleanShot 2024-11-08 at 19 05 39@2x](https://github.com/user-attachments/assets/27233a4e-7232-4ed5-833c-f649fca88e5a)  |    ![CleanShot 2024-11-08 at 18 58 17@2x](https://github.com/user-attachments/assets/f78179c3-e670-46c6-871e-666e415ca795)   |


## Why are these changes being made?


* To improve the Business Services offered

## Testing Instructions



* Apply this patch
* Navigate to the marketing tools section, i.e.  `/marketing/tools/:siteId` and `/sites/marketing/tools/:siteId`
* Make sure the designs of the banners are as expected figma design: pIiTEGIxHIzKfHv1KHOUM5-fi-3130_24773

## Pre-merge Checklist
